pages/_error.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+
+class Error extends React.Component<{ statusCode: number }> {
+    public static getInitialProps({ res, err }: { res: any; err: any }) {
```

Comment:
please use correct type for res and err (see next.d.ts res should be http.IncomingMessage maybe... give a check)